### ISSUE TITLE
Mute Sound button always flips back to an "On" state

### ISSUE THREAD

**Issue by terracoda** [phetsims/ohms-law#117]
I just noticed that the Mute Sound button keeps flipping back to an "On" state, thus I get I cannot turn the sound off. In addition, I hear both alerts,

- Sim sound off.
- Sim sound on.
 
back to back. 

This is likely a common code problem, but I found it in Ohm's Law, so posted it here.

**Comment by terracoda** [comment]
@zepumph I'll assign you, but please call in re-inforcements as needed. Tagging @jessegreenberg. 

I imagine this is not an immediate issue for Ohm's Law, but more an issue for the upcoming finalized sonified version of John Travoltage see issue https://github.com/phetsims/john-travoltage/issues/300

**Comment by jessegreenberg** [comment]
@zepumph  and I worked on this today as part of another issue, was definitely a common code problem. This button is working again, so closing this issue.
